modules/site-health/audit-enqueued-assets/helper.php

```diff
@@ -78,14 +78,29 @@ function perflab_aea_get_total_size_bytes_enqueued_styles() {
 
 /**
  * Convert full URL paths to absolute paths.
+ * Covers Standard WP configuration, wp-content outside WP directories and subdirectories.
+ * Ex: https://example.com/content/themes/, https://example.com/wp/wp-includes/
  *
  * @since 1.0.0
  *
  * @param string $resource_url URl resource link.
  * @return string Returns absolute path to the resource.
  */
 function perflab_aea_get_path_from_resource_url( $resource_url ) {
-	return ABSPATH . wp_make_link_relative( $resource_url );
+
+	// Different content folder ex. /content/.
+	if ( 0 === strpos( $resource_url, content_url() ) ) {
```

Comment:
Do we perhaps need to handle a situation where `$resource_url` is `undefined`? Maybe a:

```
if(! $resource_url ) {
  return;
}
```